WorkloadManagementSystem/DB/JobDB.sql

```diff
@@ -30,174 +30,151 @@ FLUSH PRIVILEGES;
 USE JobDB;
 
 -- ------------------------------------------------------------------------------
-DROP TABLE IF EXISTS Jobs;
-CREATE TABLE Jobs (
-    JobID INTEGER NOT NULL AUTO_INCREMENT,
-    JobType VARCHAR(32) NOT NULL DEFAULT 'normal',
-    INDEX (JobType),
-    DIRACSetup VARCHAR(32) NOT NULL,
-    INDEX (DIRACSetup),
-    JobGroup VARCHAR(32) NOT NULL DEFAULT 'NoGroup',
-    INDEX (JobGroup),
-    JobSplitType ENUM ('Single','Master','Subjob','DAGNode') NOT NULL DEFAULT 'Single',
-    INDEX (JobSplitType),
-    MasterJobID INTEGER NOT NULL DEFAULT 0,
-    Site VARCHAR(100) NOT NULL DEFAULT 'ANY',
-    INDEX (Site),
-    JobName VARCHAR(128) NOT NULL DEFAULT 'Unknown',
-    Owner VARCHAR(32) NOT NULL DEFAULT 'Unknown',
-    INDEX (Owner),
-    OwnerDN VARCHAR(255) NOT NULL DEFAULT 'Unknown',
-    INDEX (OwnerDN),
-    OwnerGroup varchar(128) NOT NULL DEFAULT 'lhcb_user',
-    INDEX (OwnerGroup),
-    SubmissionTime DATETIME,
-    RescheduleTime DATETIME,
-    LastUpdateTime DATETIME,
-    StartExecTime DATETIME,
-    HeartBeatTime DATETIME,
-    EndExecTime DATETIME,
-    Status VARCHAR(32) NOT NULL DEFAULT 'Received',
-    INDEX (Status),
-    INDEX (Status,Site),
-    MinorStatus VARCHAR(128) NOT NULL DEFAULT 'Initial insertion',
-    INDEX (MinorStatus),
-    ApplicationStatus VARCHAR(255) NOT NULL DEFAULT 'Unknown',
-    INDEX (ApplicationStatus),
-    ApplicationNumStatus INTEGER NOT NULL DEFAULT 0,
-    CPUTime FLOAT NOT NULL DEFAULT 0.0,
-    UserPriority INTEGER NOT NULL DEFAULT 0,
-    SystemPriority INTEGER NOT NULL DEFAULT 0,
-    RescheduleCounter INTEGER NOT NULL DEFAULT 0,
-    VerifiedFlag  ENUM ('True','False') NOT NULL DEFAULT 'False',
-    DeletedFlag  ENUM ('True','False') NOT NULL DEFAULT 'False',
-    KilledFlag ENUM ('True','False') NOT NULL DEFAULT 'False',
-    FailedFlag ENUM ('True','False') NOT NULL DEFAULT 'False',
-    ISandboxReadyFlag ENUM ('True','False') NOT NULL DEFAULT 'False',
-    OSandboxReadyFlag ENUM ('True','False') NOT NULL DEFAULT 'False',
-    RetrievedFlag ENUM ('True','False') NOT NULL DEFAULT 'False',
-    AccountedFlag ENUM ('True','False','Failed') NOT NULL DEFAULT 'False',
-    PRIMARY KEY (JobID)
-) ENGINE = InnoDB;
+DROP TABLE IF EXISTS `Jobs`;
+CREATE TABLE `Jobs` (
+  `JobID` INT(11) UNSIGNED NOT NULL DEFAULT 0,
```

Comment:
Yes it was dropped on purpose: the "main" table is in fact JobJDLs (where AUTO_INCREMENT on JobID is maintained), not Jobs, where we have that FOREIGN KEY (`JobID`) REFERENCES `JobJDLs`(`JobID`),